tests/acceptance/item-inline-edit-test.js

```diff
@@ -0,0 +1,216 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import '../factories/orders_package';
+import '../factories/designation';
+import '../factories/item';
+import '../factories/location';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+import { mockFindAll } from 'ember-data-factory-guy';
+
+var App, pkg;
+
+module('Acceptance: Item inline edit', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+    var location = FactoryGuy.make("location");
+    pkg = FactoryGuy.make("item", { id: 50, state: "submitted" , quantity: 10, height: 10, width: 15, length: 20, notes: "Inline edit test" });
+    mockFindAll('location').returns({json: {locations: [location.toJSON({includeId: true})]}});
```

Comment:
@trojanh , as I was testing with Height, Width and Length parameters by changing their values I needed them to add here and they are also set as default parameters in Factory itself so any user can use them.
About the line break, yeah it is a good Idea but through out our project we have followed this convention so I followed it and also if it is upto 100 lines than it is fine.